lib/matplotlib/backends/__init__.py

```diff
@@ -14,6 +16,49 @@
     if not line.startswith('  File "<frozen importlib._bootstrap'))
 
 
+def _get_current_event_loop():
```

Comment:
This should be something like `_get_interactive_framework_name`.  `_get_current_event_loop` makes me think it will return to me an object I can use as an event loop (ala `asyncio.get_event_loop()`).

Reply:
Done, see below.